spec/src/main/asciidoc/required-apis.asciidoc

```diff
@@ -46,83 +46,79 @@ MicroProfile applications that make use of APIs or versions not mentioned below
-Use of implementations beyond JAX-RS 2.1 in MicroProfile is allowed, however, not required as of version 2.0 of MicroProfile.
+JAX-RS provides both standard client and server APIs for RESTful communication by MicroProfile 4.0 applications.
+Use of implementations beyond JAX-RS 2.1 in MicroProfile is allowed, however, not required as of version 4.0 of MicroProfile.
 
- - https://github.com/jax-rs
- - https://jcp.org/en/jsr/detail?id=370
+ - https://jakarta.ee/specifications/restful-ws/2.1/restful-ws-spec-2.1.html
+ - https://github.com/eclipse-ee4j/jaxrs-api
 
-[[javaee-jsonb]]
-=== JSON-B 1.0
+[[jakarta-jsonb]]
+=== Jakarta JSON-B 1.0
 
-JSON-B is leveraged as part of MicroProfile 2.0 to provide standard APIs for binding JSON documents to Java code.
-Use of implementations beyond JSON-B 1.0 in MicroProfile is allowed, however, not required as of version 2.0 of MicroProfile.
+JSON-B is leveraged as part of MicroProfile 4.0 to provide standard APIs for binding JSON documents to Java code.
+Use of implementations beyond JSON-B 1.0 in MicroProfile is allowed, however, not required.
 
- - https://javaee.github.io/jsonb-spec
- - https://jcp.org/en/jsr/detail?id=367
+ - https://jakarta.ee/specifications/jsonb/1.0/jsonb-spec-1.0.html
+ - https://github.com/eclipse-ee4j/jsonb-api
 
-[[javaee-jsonp]]
-=== JSON-P 1.1
+[[jakarta-jsonp]]
+=== Jakarta JSON-P 1.1
 
-JSON-P is leveraged as part of MicroProfile 2.0 to provide standard APIs for processing JSON documents.
-Use of implementations beyond JSON-P 1.1 in MicroProfile is allowed, however, not required as of version 2.0 of MicroProfile.
+JSON-P is leveraged as part of MicroProfile 4.0 to provide standard APIs for processing JSON documents.
+Use of implementations beyond JSON-P 1.1 in MicroProfile is allowed, however, not required.
 
- - https://javaee.github.io/jsonp/
- - https://jcp.org/en/jsr/detail?id=374
+ - https://jakarta.ee/specifications/jsonp/1.1/jsonp-spec-1.1.html
+ - https://github.com/eclipse-ee4j/jsonp
 
-[[javaee-common-annotations]]
-=== Common Annotations 1.3
+[[jakarta-annotations]]
+=== Jakarta Annotations 1.3
 
-Common Annotations provides annotations for common semantic concepts across a variety of individual technologies in the Java(TM) SE and Java(TM) EE platforms.
-Use of implementations beyond Common Annotations 1.3 in MicroProfile is allowed, however, not required as of version 2.0 of MicroProfile.
+Jakarta Annotations provides annotations for common semantic concepts across a variety of individual technologies in the Java(TM) SE and Jakarta EE platforms.
+Use of implementations beyond Jakarta Annotations 1.3 in MicroProfile is allowed, however, not required.
 
- - https://javaee.github.io/glassfish/
- - https://jcp.org/en/jsr/detail?id=250
+ - https://jakarta.ee/specifications/annotations/1.3/annotations-spec-1.3.html
+ - https://github.com/eclipse-ee4j/common-annotations-api
 
 [[mp-config]]
-=== Eclipse MicroProfile Config 1.4
+=== Eclipse MicroProfile Config 2.0
```

Comment:
When MP 4.0 wil be released after the WG has formed, it is no longer necessary to call it 'Eclipse MicroProfile', only just 'MicroProfile', no?

Reply:
Correct.  But, we can't make that change yet... For the Mx releases, I think we need to stick with the Eclipse MicroProfile wording...

Reply:
Agreed.